engines/python/src/main/java/ai/djl/python/engine/PyEnv.java

```diff
@@ -321,6 +342,24 @@ public int getTensorParallelDegree() {
         return tensorParallelDegree;
     }
 
+    /**
+     * Returns the pipeline parallel degree.
+     *
+     * @return the pipeline parallel degree
+     */
+    public int getPipelineParallelDegree() {
```

Comment:
Can you remove the pipelineParallelDegree read from this PR as we're not directly using it here. I will add this separately later as required

Reply:
Removed.